app/src/main/java/org/gdg/frisbee/android/onboarding/AppInviteLinkGenerator.java

```diff
@@ -63,4 +78,30 @@ private String createDeepLink(String gplusId) {
             .toString();
     }
 
+    private void shareShortAppInviteLink(String longUrl) {
+        FirebaseDynamicLinksHub firebaseDynamicLinksHub = App.from(activity).getFirebaseDynamicLinksHub();
+        String requestString = "{\"longDynamicLink\": \"" + longUrl + "\",\"suffix\": {\"option\": \"SHORT\"}}";
+        RequestBody body = RequestBody.create(okhttp3.MediaType.parse("application/json; charset=utf-8"), requestString);
+        firebaseDynamicLinksHub.getShortenedUrl(BuildConfig.FIREBASE_WEB_API_KEY, body).enqueue(new Callback<FirebaseDynamicLinksResponse>() {
+            @Override
+            public void onSuccess(FirebaseDynamicLinksResponse response) {
+                createChooser(HttpUrl.parse(response.getShortLink()));
+                shareProgressDialog.dismiss();
```

Comment:
I didn't have time to check the latest version of the PR. Sorry. 
Let's schedule some time and talk about it. Can also be a chat or Hangouts video call.